src/current/_includes/releases/v24.2/v24.2.4.md

```diff
@@ -0,0 +1,95 @@
+- Fixed a bug where zone configuration changes issued by the [declarative schema changer]({% link v24.2/online-schema-changes.md %}#declarative-schema-changer) were not blocked if a table had `schema_locked` set. [#130705][#130705]
+- Fixed a bug in which some [`SELECT FOR UPDATE`]({% link v24.2/select-for-update.md %}) or [`SELECT FOR SHARE`]({% link v24.2/select-for-update.md %}) queries using `NOWAIT` could still block on locked rows when using the `optimizer_use_lock_op_for_serializable` [session setting]({% link v24.2/session-variables.md %}) under [`SERIALIZABLE`]({% link v24.2/demo-serializable.md %}) isolation. This bug was introduced with [`optimizer_use_lock_op_for_serializable`]({% link v24.2/session-variables.md %}#optimizer-use-lock-op-for-serializable) in v23.2.0. [#130430][#130430]
+- Fixed a bug that caused the [optimizer]({% link v24.2/cost-based-optimizer.md %}) to plan unnecessary post-query uniqueness checks during [`INSERT`]({% link v24.2/insert.md %}), [`UPSERT`]({% link v24.2/upsert.md %}), and [`UPDATE`]({% link v24.2/update.md %}) statements on tables with [partial]({% link v24.2/partial-indexes.md %}), [unique]({% link v24.2/create-index.md %}#unique-indexes) [hash-sharded indexes]({% link v24.2/hash-sharded-indexes.md %}). These unnecessary checks added overhead to execution of these statements, and caused the statements to error when executed under [`READ COMMITTED`]({% link v24.2/read-committed.md %}) isolation. [#130570][#130570]
+- Fixed a bug that could result in the inability to garbage collect an [MVCC]({% link v24.2/architecture/storage-layer.md %}#mvcc) range tombstone within a [global table]({% link v24.2/global-tables.md %}). [#130940][#130940]
+- Previously, if a connection was attempting a [schema change]({% link v24.2/online-schema-changes.md %}) while the same schema objects were being dropped, it was possible for the connection to be incorrectly dropped. This is now fixed. [#130962][#130962]
+- Previously, CockroachDB could incorrectly evaluate the [`IS NOT NULL`]({% link v24.2/null-handling.md %}#nulls-and-simple-comparisons) when applied to tuples containing NULL elements (e.g., `(1, NULL)` or `(NULL, NULL)`). This issue, which has existed since version 20.2, has now been resolved to ensure that only fully non-NULL tuples pass the filter. [#130947][#130947]
+- Fixed a bug that could cause errors with the message `internal error: Non-nullable column ...` when executing statements under [`READ COMMITTED`]({% link v24.2/read-committed.md %}) isolation that involved tables with [`NOT NULL`]({% link v24.2/not-null.md %}) virtual columns. [#131019][#131019]
+- The AWS endpoint and cloud custom HTTP client configuration are now considered when [implicit authentication]({% link v24.2/cloud-storage-authentication.md %}) is used for cloud storage. Previously, these were only considered when using explicit credentials. [#131173][#131173]
```

Comment:
I have the same note in v23.2.13 and here is my edit. Which do you prefer?

```
- Fixed a bug where [AWS S3 and HTTP client configurations]({% link v23.2/cloud-storage.md %}) were not considered when implicit authentication was used. [#131201][#131201]
```

Reply:
```suggestion
- Fixed a bug where [AWS S3 and HTTP client configurations]({% link v24.2/cloud-storage.md %}) were not considered when implicit authentication was used. [#131173][#131173]
```